### ISSUE TITLE
[Bug] "Unsynced" Action Modifier not working

### ISSUE THREAD

**Issue by Katsuvera** [ffxivcode/AutoDuty#358]
### What happened?

As mentioned before, Actions that are labeled as "Unsynced" are still occurring even while Synced. Happening over multiple runs in Sohm Al and The Vault so seems consistently broken.

### Version Number

144

### Steps to reproduce the error

Enter The Vault with any of the paths while Synced and reach the first boss. Or make your own test path with the first step being UnsyncedMoveTo is probably faster.

**Comment by Herculezz55** [comment]
question? are you unsyncing manually or having AD do it? cause it currently is only checking the Configuration.Unsyned, if you are doing it manually in game that would be why its no skipping them

**Comment by Katsuvera** [comment]
Oh, okay, I believe I found the issue.

What happened was:
1.) I did one Unsynced run via AD of some random dungeon. In other words, I went to "Regular" Duty Mode, checked the "Unsynced" box, and ran whatever on my DNC.

2.) I then went back to running normal, Synced Duty Support stuff on my other classes. aka. I chose "Support" and started running Sohm Al with Duty Support.

It seems the issue lies in the fact that I _did not uncheck the "Unsynced" option_ before switching to the "Support" option in AD and running other stuff. It was because of this that AD was _not_ skipping the steps with Unsynced in front. Presumably because that "Unsynced" button was still selected in the background. I say "in the background" cuz you can't see it unless you're still on the "Regular" Duty Mode option.

TL;DR I was unaware that I needed to un-select the "Unsynced" option myself before switching out of "Regular" Duty Mode".

With that said, I must admit that I don't understand where the "if you are doing it manually in game" part you mentioned comes in to play. I manually selected the game's "Unrestricted Party" option in the game's Duty Finder Settings but it didn't seem to affect AD skipping the Unsynced steps in my Synced runs. As far as I can tell, it's just the Unsynced checkbox in AD.

**Comment by Herculezz55** [comment]
OK that makes sense, I will have it also check what mode u are on.

**Comment by Katsuvera** [comment]
That would be great! Thank you very much. :D
